xmtp_mls/src/storage/encrypted_store/schema.rs

```diff
@@ -107,6 +107,13 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    user_preferences (id) {
+        id -> Nullable<Integer>,
```

Comment:
Question: is it correct to be nullabe?

Reply:
Yes, it's correct. We want the database to provide this field if it's null.